### DISCUSSION TITLE
Welcome to netbox-dns Discussions!

### DISCUSSION THREAD

**Discussion by hbasria** [auroraresearchlab/netbox-dns#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peteeckel** [comment]
Oops ... found this only now, when I looked the last time it wasn't there yet :-)

Quick introduction: I've been working as a freelance consultant since 1994. so long I had to start to prune my CV so it didn't look like a book of ancient history (which in fact it did, containing items such as VMS, X.500, DECnet/OSI and other long forgotten things).

So currently I'm mainly working in Monitoring projects (Icinga, Grafana, Graylog and the like), Automation (mainly Ansible), System integration (everything that doesn't make it up the next tree is game) and general Linux stuff ... and, recently, NetBox.

I became aware of the netbox-dns Plugin just after I decided to implement something like it myself for a customer project, just in time to save me the tedious work of starting from scratch - thanks for that! Since some essential parts are still missing I started implementing some functionality, learning some NetBox internals and Django on the fly - I'm rather impressed by both. 

As soon as the requirements for the customer project (a plain vanilla one-zone DNS environment, but they need something to maintain their records and zones without writing zone files by hand) are met, I'm planning to drive it a bit further with a multi-view environment for my own infrastructure.

Currently the community seems to be quite small, but once the plugin can actually be used to manage complete DNS environments I'm quite sure it will grow quickly - I'm looking forward to that!

**Comment by LuPo** [comment]
Hello :wave: .

Let me also introduce myself showing support and hoping that this community can grow. I do appreciate your work on this project and hope I can also contribute back a little bit.
I've been working since 2007 mainly as infrastructure/network engineer in a public research institution in Italy and currently as a Solutions Architect at a South African university. My main interests are Networks and Automation. Although I do not have a programming formal education I fell in love with Python and want to get up to speed with its usage. At the university, we are strongly relying on NetBox and want to integrate more processes that use it as a source of truth while building device and service configurations. Automating DNS is a next big step on the journey and netbox-dns can be the perfect tool to source configuration variables for server deployments (mainly through Ansible) and input/store the zone records. To operationalize the entire process we are investigating octoDNS to synchronize the RRs between different DNS providers while validating the changes through the Git-based workflows.
